python/grizzly/grizzly/grizzly.py

```diff
@@ -116,21 +250,10 @@ def __setitem__(self, key, value):
                 key
             )
 
-    def __getattr__(self, key):
```

Comment:
This function needs to be deleted. Otherwise, we cannot access any of the member variables of the class.